File: B/PHP/Processor.php
<?php

// Kelas Processor
class Processor
{
    private $name;
    private $price;

    // Constructor
    function __construct($name, $price) 
    {
        $this->name = $name;
        $this->price = $price;
    }

    // Metode Setter
    function setName($name) 
    {
        $this->name = $name;
    }

    function setPrice($price) 
    {
        $this->price = $price;
    }

    // Metode Getter
    function getName() 
    {
        return $this->name;
    }

    function getPrice() 
    {
        return $this->price;
    }

    // Metode Print
    function printProcessor() 
    {
        echo "===============================". "<br/>";
        echo "Prosessor:". "<br/>";
        echo "===============================". "<br/>";
        echo "Nama      : ". $this->getName(). "<br/>";
        echo "Harga     : ". "Rp". $this->getPrice(). "<br/>";
    }
}
?>

File: B/Python/Processor.py
# Kelas Processor
class Processor:
    __name = ""
    __price = 0

    # Constructor
    def __init__(self, name="", price = 0):
        self.__name = name
        self.__price = price

    # Metode Setter
    def setName(self, name):
        self.__name = name

    def setPrice(self, price):
        self.__price = price

    # Metode Getter
    def getName(self):
        return self.__name

    def getPrice(self):
        return self.__price
    
    # Metode Print
    def printProcessor(self):
        print("===============================")
        print("Prosessor:")
        print("===============================")
        print("Nama         : " + self.getName())
        print("Harga        : " + "Rp" + str(self.getPrice()))